notify/grafana_alertmanager.go

```diff
@@ -469,14 +469,14 @@ func newTestReceiversResult(alert types.Alert, results []result, receivers []*AP
 func TestReceivers(
 	ctx context.Context,
 	c TestReceiversConfigBodyParams,
-	tmpls []string,
+	tmpls []templates.TemplateDefinition,
 	buildIntegrationsFunc func(*APIReceiver, *template.Template) ([]*nfstatus.Integration, error),
 	externalURL string) (*TestReceiversResult, int, error) {
 
 	now := time.Now() // The start time of the test
 	testAlert := newTestAlert(c, now, now)
 
-	tmpl, err := templateFromContent(tmpls, externalURL)
+	tmpl, err := templates.TemplateFromTemplateDefinitions(tmpls, log.NewNopLogger(), externalURL)
```

Comment:
NIT: Probably fine to add a `logger log.Logger` parameter to this function. It's only external use is from Mimir's `func (am *Alertmanager) TestReceiversHandler` which has a logger handy to pass in.